ycms/static/src/js/ward/discharge.ts

```diff
@@ -0,0 +1,103 @@
+import { Modal, ModalOptions } from "flowbite";
+
+let selectedBedId: string | null = "";
+let selectedBedAssignmentId: string | null = "";
+
+export const handleModalClose = (): void => {
+    document.getElementById("cancelDischargeButton")?.addEventListener("click", () => {
+        const modal = (FlowbiteInstances as any).getInstance("Modal", "modalEl");
+        modal.hide();
+    });
+    document.getElementById("closeModalButton")?.addEventListener("click", () => {
+        const modal = (FlowbiteInstances as any).getInstance("Modal", "modalEl");
```

Comment:
```suggestion
        const modal = FlowbiteInstances.getInstance("Modal", "modalEl");
```
not necesary, and eslint is complaining about `explicit-any` otherwise